src/chpasswd.c

```diff
@@ -392,12 +392,49 @@ static void close_files (void)
 	pw_locked = false;
 }
 
+static const char *get_salt(void)
+{
+	if (   !eflg
+		&& (   (NULL == crypt_method)
+			|| (0 != strcmp (crypt_method, "NONE")))) {
```

Comment:
As you are changing this part of the code. Can you make it easier to read?

Reply:
I have not changed the code. I just moved this part of code in a function.

Reply:
This is more about the spaces/tab between `if (` and `!eflg`. I don't know why the original author did that but IMO it's difficult to read. Can you change it to something like:
```
	if (!eflg
	    && ((NULL == crypt_method) || (0 != strcmp (crypt_method, "NONE")))) {
```

This is the best I could come up with, but I am open to better solutions.

Reply:
> This is more about the spaces/tab between `if (` and `!eflg`. I don't know why the original author did that but IMO it's difficult to read. Can you change it to something like:
> 
> ```
> 	if (!eflg
> 	    && ((NULL == crypt_method) || (0 != strcmp (crypt_method, "NONE")))) {
> ```
> 
> This is the best I could come up with, but I am open to better solutions.

OK, I have refactored this function